tools/rna_tools/ribotaper/tool_dependencies.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="samtools" version="0.1.19">
+        <repository name="package_samtools_0_1_19" owner="devteam" />
```

Comment:
sorry, this is not the final version. the dependency is according to conda now. I will update the branch soon.